src/test/java/MariaBassTest.java

```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class MariaBassTest {
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://askomdch.com/");
+    }
+
+    @AfterMethod
+    public void afterMethod() {
+        driver.quit();
+    }
+
+    @Test
+    public void checkSaleTag_TC_001() {
```

Comment:
The name of the test must contain the word "test"

Reply:
Ok, thank you!